scripts/pre_commit_linter.py

```diff
@@ -490,38 +490,38 @@ def _get_all_files(input_path, input_files):
             print '----------------------------------------'
             sys.exit(1)
         if os.path.isfile(input_path):
-            all_files = [input_path]
+            all_filenames = [input_path]
```

Comment:
Here and elsewhere, do you actually mean get_all_filepaths?

"filename" is just the filename, like `pp.mp3`. "filepath" includes the stuff before the filename, like `this/is/a/dir/pp.mp3`.

Reply:
Done
As we are dealing with filepaths in the pre_commit_linter.py, so I have changed all filename instances with filepath or filepaths as suitable.